cloudinit/config/cc_ssh_import_id.py

```diff
@@ -18,6 +18,7 @@
 # https://launchpad.net/ssh-import-id
 distros = ["ubuntu", "debian"]
 
+SSH_IMPORT_ID = "ssh-import-id"
```

Comment:
Nit: Something like SSH_IMPORT_ID_BINARY instead? Without seeing where it gets used, I wouldn't know why this variable exists.